collection-scripts/common.sh

```diff
@@ -15,3 +15,28 @@ function get_operator_ns() {
         exit 1
     fi
 }
+
+get_log_collection_args() {
+	LOG_COLLECTION_ARGS=""
+
+	if ! [ -z "${MUST_GATHER_SINCE+x}" ]; then
+		LOG_COLLECTION_ARGS=--since="${MUST_GATHER_SINCE}"
```

Comment:
Since must-gather command in oc is the caller and validation is done here, we don't have to validate that MUST_GATHER_SINCE and MUST_GATHER_SINCE_TIME are both set.

Reply:
Just a note for the future reviewers :)

Reply:
Ah yes, that is a good note to make. Added a comment to the code (will push soon)